executor/common_kvm.h

```diff
@@ -1,6 +1,9 @@
 // Copyright 2025 syzkaller project authors. All rights reserved.
 // Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
+#ifndef EXECUTOR_COMMON_KVM_H
```

Comment:
The original idea was to not include headers multiple times. But I guess we can do this if carefully remove guards.